### PR TITLE
Improve 2FA auth UI

### PR DESCRIPTION
**Pull request by dennisreimann**
Updates for the views involved in 2FA, based on @Kukks' recent updates.

### CODE REVIEW

**Comment by pavlenex** [comment]
tACK.

A nit, but I think the button on the login page should be a bit more prominent, it looks like  secondary button and one may miss at first glance that they'd have to click on it to proceed.

Screenshot 2022-01-18 at 10 44 11

**Comment by dennisreimann** [comment]
Linkcheck test failure unrelated.

File: BTCPayServer/Views/UIAccount/LoginWithFido2.cshtml [hunk 1]
```diff
@@ -10,22 +10,23 @@
...
-            <p>Insert your security key into your computer's USB port. If it has a button, tap on it.</p>
+        <p>Insert your security device and proceed.</p>
+        <div id="info-message" class="alert alert-info mb-3 d-none">
+            <div class="d-flex align-items-center">
+                <span id="spinner" class="fa fa-spinner fa-spin me-3 mt-1 fido-running" style="font-size:1.5rem"></span>
+                <span>If your security device has a button, tap on it.</span>
+            </div>
         </div>
-        <button id="btn-start" class="btn btn-outline-primary w-100 btn-lg d-none" type="button">Start</button>
+        <button id="btn-start" class="btn btn-outline-primary d-none" type="button">Start</button>
```
Comment (dennisreimann) [inline BTCPayServer/Views/UIAccount/LoginWithFido2.cshtml:20]:
```suggestion
        <button id="btn-start" class="btn btn-primary d-none" type="button">Start</button>
```

**Comment by dennisreimann** [comment]
@pavlenex Good catch, updated!

**Review by Kukks — APPROVED** [review]

**Pull request merged**